source/linux/Foundational_Components/Power_Management/pm_low_power_modes.rst

```diff
@@ -290,6 +291,126 @@ system and it will go through a normal Linux boot process.
+   #. Better efficiency: I/O Only Plus DDR mode can help to improve the efficiency of embedded devices by
+      reducing the amount of time that the processor is idle. This is because the processor can
+      be kept in a low-power state when it is not needed.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. important:: Jumper J9 should be connected on SK to enable system to enter I/O Only plus DDR mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. important:: Jumper J12 should be connected on SK to enable system to enter I/O Only plus DDR mode.
+
+   The wakeup sources that can be used to wake the system from I/O Only Plus DDR are
+   mcu_uart0, mcu_mcan0, mcu_mcan1 and wkup_uart0. After Linux boots, direct register
+   writes can be used to enable wakeup.
+
+   .. rubric:: Following commands can be used to enable wakeup from mcu_mcan0, mcu_mcan1,
+               mcu_uart0 and wkup_uart0 respectively.
+
+   .. code-block:: console
+
+      root@<machine>:~# devmem2 0x4084038 0x20050000
+      root@<machine>:~# devmem2 0x4084040 0x20050000
+      root@<machine>:~# devmem2 0x4084014 0x20050000
+      root@<machine>:~# devmem2 0x4084024 0x20050000
+
+   .. note::
+
+      Atleast one of the wakeup sources listed above must be enabled to wakeup from I/O Only Plus DDR mode.
+
+   .. rubric:: To enter I/O Only Plus DDR mode, first disable wakeup from RTC, USB0 and USB1 as these wakeup
+               sources are not supported for this mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. code-block:: console
+
+         root@am62axx-evm:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
+         root@am62axx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f900000.dwc3-usb/power/wakeup
+         root@am62axx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f910000.dwc3-usb/power/wakeup
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. code-block:: console
+
+         root@<machine>:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
+         root@<machine>:~# echo disabled > /sys/devices/platform/bus@f0000/f900000.usb/power/wakeup
+         root@<machine>:~# echo disabled > /sys/devices/platform/bus@f0000/f910000.usb/power/wakeup
+
+   .. rubric:: Then, configure PMIC register bit to turn off only selected rails for this mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. code-block:: console
+
+         root@am62axx-evm:~# i2cset -f -y -m 0xFF -r -a 0 0x48 0x86 0x1
+
+      The register write has been done to enable PMIC to enter `PMIC S2R <https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.ti.com/lit/pdf/slvucm3&ved=2ahUKEwjknrL864-KAxVocGwGHdJZFPwQFnoECBgQAQ&usg=AOvVaw1y0evgQxxy9YOF7l5Wzaef>`_ .
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. code-block:: console
+
+         root@am62axx-evm:~# i2cset -f -y -m 0xFF -r -a 0 0x48 0x86 0x2
```

Comment:
```suggestion
         root@am62pxx-evm:~# i2cset -f -y -m 0xFF -r -a 0 0x48 0x86 0x2
```

Since it's under CONFIG_part_variant in ('AM62PX')

Reply:
Agree. Updated. Thanks